data/sql/updates/pending_db_world/rev_1689005068014101100.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_auth` (`sql_rev`) VALUES ('1689005171898848000');
```

Comment:
this should not be here If I remember correctly, indeed the error in the github pipeline is the following 
```
ERROR 1146 (42S02) at line 2 in file: '/home/runner/work/azerothcore-wotlk/azerothcore-wotlk/data/sql/updates/db_world/2023_07_10_03.sql': Table 'acore_world.version_db_auth' doesn't exist
```
```suggestion
```